commerce-product-test-util/src/main/java/com/liferay/commerce/product/test/util/CPTestUtil.java

```diff
@@ -138,6 +148,63 @@ public static CPInstance addCPInstance(long groupId)
 			cpDefinition.getCPDefinitionId(), CPInstanceConstants.DEFAULT_SKU);
 	}
 
+	public static CPInstance addCPInstanceWithSku() throws PortalException {
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext();
+
+		User user = UserLocalServiceUtil.getUser(serviceContext.getUserId());
+
+		CPDefinition cpDefinition = _addCPDefinition(
+			SimpleCPTypeConstants.NAME, true, true, serviceContext);
+
+		long now = System.currentTimeMillis();
+
+		Date displayDate = new Date(now - Time.HOUR);
+		Date expirationDate = new Date(now + Time.DAY);
+
+		Calendar displayCal = CalendarFactoryUtil.getCalendar(
```

Comment:
Variable name should be `displayCalendar` spelled out (rather than abbreviated to `displayCal`)